client/lib/plans-list/index.js

```diff
@@ -96,15 +97,13 @@ PlansList.prototype.initialize = function( plans ) {
 };
 
 /**
- * Parse data returned from the API
+ * Parses data retrieved from the API and extracts the list of plans.
  *
- * @param {array} data
- * @return {array} plans
+ * @param {object} data - raw data
+ * @return {object} a list of plans
  **/
 PlansList.prototype.parse = function( data ) {
-	delete data._headers;
-
-	return data;
+	return omit( data, '_headers' );
```

Comment:
This endpoint returns an array, to which a `_headers` property is assigned:

![screen shot 2016-03-16 at 2 52 11 pm](https://cloud.githubusercontent.com/assets/1130674/13829902/bc329e5a-eb86-11e5-9e35-75375040cdce.png)

_.omit returns an object, but to preserve the behavior of this function, which currently returns an array, we should use _.reject here.